_episodes/33.md

```diff
@@ -0,0 +1,266 @@
+ ([@siddhant-chadha](https://www.linkedin.com/in/siddhant-chadha-838136142/)).
+ * Suman Baliganahalli Narayan Murthy, Vice President at [Goldman Sachs](https://developer.gs.com/discover/home)
+ ([@suman-b-n](https://www.linkedin.com/in/suman-b-n-08-03-1990/)).
+ * Sumit Halder, Vice President at [Goldman Sachs](https://developer.gs.com/discover/home)
+  ([@sumit-halder](https://www.linkedin.com/in/sumit-halder-a3732482/)).
+ 
+## Releases 369, 370, and 371
+
+Trino 369
+
+* Experimental support for task level retries.
+* Support for groups in OAuth2 claims.
+* Column comments in ClickHouse connector.
+* Write Bloom filters in ORC files.
+* Procedure for optimizing Iceberg tables.
+
+Trino 370
+
+* Add CLI support for ARM64
+* Improved performance for ORC
+* Improved performance for map and row types
+* Reduced latency for OAuth2.0 authentication
+
+Trino 371
+
+* Support for secrets and user group selector in resource group manager.
+* Support AWS role session name in S3 security mapping configuration.
+* Many bug fixes.
+
+Notes from Manfred
+
+* Add support for using PostgreSQL and Oracle as backend database for resource
+  groups.
+* Remove `spill-order-by`,  `spill-window-operator`, and
+  `query.max-total-memory-per-node`.
+* Add support for `ALTER MATERIALIZED VIEW ... SET PROPERTIES` in the engine.
+* Prevent hanging query execution on failures with phased execution policy.
+* Support for renaming schemas in PostgreSQL and Redshift connectors.
+* Lots of improvements on Clickhouse connector, thanks Yuya!
+* Update to newer ClickHouse version removed support for Altinity 20.3
+* `$properties` table and other hidden tables in Iceberg connector, including
+  docs.
+* Automatically adjust `ulimit` setting when using the RPM package.
+* Docker images changes to UBI.
+* Remove support/need for `allow-drop-table` catalog property in JDBC connectors.
+* A bunch of SPI changes
+* DML with Iceberg connector with fault tolerant mode and more Tardigrade improvements
+* Drop support for Kudu 1.13.0
+  
+More detailed information is available in the [Trino
+369](https://trino.io/docs/current/release/release-369.html), [Trino
+370](https://trino.io/docs/current/release/release-370.html), and [Trino
+371](https://trino.io/docs/current/release/release-371.html) release notes.
+
+## Concept of the month: High Availability with Trino
+
+Goldman Sachs uses Trino to reduce last-mile ETL and provide a unified way of 
+accessing data through federated joins. Making a variety of data sets from 
+different sources available in one spot for our data science team was a tall 
+order. Data must be quickly accessible to data consumers and systems like Trino
+must be reliable for users to trust this singular access point for their data.
+
+In order for analysts and data scientists to use these services, they first need
+to trust in the system. It was vital to Goldman Sachs that Trino have High
```

Comment:
Trino has high availability. In the event of any failure, another Trino cluster is available to process requests.